mongodb.py

```diff
@@ -90,7 +117,7 @@ def do_server_status(self):
         #globalLocks
         if 'globalLock' in server_status:
             if 'totalTime' in server_status['globalLock']:
-                self.submit('globalLock_totalTime', server_status['globalLock']['totalTime'])
+                self.submit('globalLock_time','total', server_status['globalLock']['totalTime'])
```

Comment:
Make all all globalLock type and put time,currentQueue,activeClients into type_instance